nginx/nginx.template

```diff
@@ -90,6 +90,11 @@ http {
       proxy_pass ${BEDTIME_URL};
     }
 
+    # Proxy social
+    location /social/ {
+      proxy_pass http://social:8001/;
```

Comment:
maybe remove trailing `/`

Reply:
for both location and after the port number

Reply:
unfortunately this is necessary -- it's how nginx strips /social/ from the forwarded url

Reply:
https://stackoverflow.com/questions/41605137/how-to-remove-a-subdirectory-from-the-url-before-passing-to-a-script

Reply:
better one https://serverfault.com/questions/699695/nginx-proxy-pass-remove-url-substring